nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ControlRate.java

```diff
@@ -71,30 +70,57 @@ public class ControlRate extends AbstractProcessor {
     public static final String DATA_RATE = "data rate";
```

Comment:
Small issue but I would like to see these capitalized (Data Rate) rather than all lower case as is currently the case. Not sure if this will impact existing flows - if that's the case, leave it as is.

Reply:
I believe this is a breaking change that would affect existing flows. At least, when I updated the values to have a different case, e.g. "Data Rate", all unit tests failed. I don't see a big deal with this, and to avoid potential problems I think it should be left as-is.